source/_components/cover.tahoma.markdown

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: "Tahoma Cover"
+description: "Instructions how to integrate Tahoma covers into Home Assistant."
+date: 2017-07-18 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
```

Comment:
There doesn't seem to be a logo named `tahoma.png`. Am I wrong?

Reply:
You are absolutely right!
@philklei ?